components/backends/trtllm/src/dynamo/trtllm/utils/request_handlers/handler_base.py

```diff
@@ -135,7 +135,10 @@ async def generate_locally(self, request: dict):
         ignore_eos = request["stop_conditions"]["ignore_eos"]
         if ignore_eos:
             sampling_params.ignore_eos = ignore_eos
-            sampling_params.min_tokens = max_tokens
+        
+        min_tokens = request["stop_conditions"]["min_tokens"]
+        if min_tokens:
+            sampling_params.min_tokens = min_tokens
```

Comment:
Just curious - did it work how it was before, or is splitting min_tokens out separately what made it work?